From e2b85dd054236a798daeab19f13f717aaf67828c Mon Sep 17 00:00:00 2001
From: Jialin Zhang <zhangjialin11@huawei.com>
Date: Tue, 4 Jan 2022 12:04:11 +0800
Subject: [PATCH] kabi: reserve space for arm64 cpufeature related structure
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: e2b85dd054236a798daeab19f13f717aaf67828c

hulk inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I4JBL0
CVE: NA

-------------------------------

Reserve space for the structure cpu_hwcap_keys and cpu_hwcaps.

Signed-off-by: Jialin Zhang <zhangjialin11@huawei.com>
Reviewed-by: wangxiongfeng <wangxiongfeng2@huawei.com>
Reviewed-by: Yang Yingliang <yangyingliang@huawei.com>
Reviewed-by: Hanjun Guo <guohanjun@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/arm64/include/asm/cpucaps.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/include/asm/cpucaps.h b/arch/arm64/include/asm/cpucaps.h
index 93bae3795165..c58a1919bfc9 100644
--- a/arch/arm64/include/asm/cpucaps.h
+++ b/arch/arm64/include/asm/cpucaps.h
@@ -72,6 +72,6 @@
 #define ARM64_HAS_TWED				62
 #define ARM64_WORKAROUND_HISILICON_1980005	63
 
-#define ARM64_NCAPS				64
+#define ARM64_NCAPS				80
 
 #endif /* __ASM_CPUCAPS_H */
-- 
2.26.2

